samples/word/25-paragraph/get-word-count.yaml

```
@@ -12,11 +12,13 @@ script:
         $("#count").click(() => tryCatch(run));
 
         async function run() {
+          // Count how many times each word/term appears in the document.
           await Word.run(async (context) => {
             const paragraphs = context.document.body.paragraphs;
             paragraphs.load("text");
             await context.sync();
 
+            // Split up document text using existing spaces as separator/delimiter.
```

Another suggestion to remove the '/'.
```suggestion
            // Split up document text using existing spaces as the delimiter.
```